README.md

```diff
@@ -209,15 +219,18 @@ To send wrapped Bitcoin back to Bitcoin, run the following commands:
     ```
     axelarcli tx bitcoin sign-pending-transfers 0.0001btc --gas="auto" --gas-adjustment=1.15 --from validator -b block -y
     ```
-5. Submit the transfer to Bitcoin
+
+    **Note**: If you are having gas error issue `out of gas: out of gas in location: WriteFlat; gasWanted: 200000, gasUsed: 200780"`, try to increment the `gas-adjustement` to a higher value `1.2` or `1.3`. With more deposits, the transaction become larger and hence requires more gas
```

Comment:
i would just change the command to add auto-gas flags 
`— gas="auto" — gas-adjustment=1.15`

Reply:
`--gas=auto` flag is already present, would that flag alone suffice ? I had in the past to also add the flag --gas-adjustement to get a tx signature to work

Reply:
ah, i didn't see that we already updated the command. with auto flags.